packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionStatusInfo.tsx

```diff
@@ -69,23 +94,15 @@ export const TransactionStatusInfo = ({
             } ready to claim`}
           </span>
         ) : null}
-        <span className="-ml-1">.</span>
+        <span>.{` `}</span>
+        {/* open tx history panel cta */}
+        <ExternalLink className="arb-hover text-sm underline">
+          <span className="hidden lg:inline">
+            Open Transaction History panel.
+          </span>
+          <span className="lg:hidden">Show.</span>
```

Comment:
I don't think we should change the text to `Show.`
Reasons:
- `Show.` is less clear than the original text
- `Show.` is an incomplete sentence and should not have a full stop. It feels wrong.
- `Show` can be a button but we are not using a button here

Why isn't this implemented as on mockup? (Place `Open....` on the second line)